core/exchange_controller.go

```diff
@@ -61,22 +61,10 @@ func CalculateBetaFromMiningChoiceAndConversions(hc *HeaderChain, block *types.W
 
 	// Firstly calculated the new beta from the best diff calculated from the previous step
 	// Since, -B0/B1 = diff/log(diff), B1 is set to 1
-	newBeta0 := new(big.Float).Quo(new(big.Float).SetInt(bestDiff), new(big.Float).SetInt(common.LogBig(bestDiff)))
-	newBeta0 = new(big.Float).Mul(newBeta0, big.NewFloat(-1))
+	newBeta0OverBeta1 := new(big.Int).Div(new(big.Int).Mul(bestDiff, common.Big2e64), common.LogBig(bestDiff))
 
-	// convert the beta values into the big numbers so that in the exchange rate
-	// computation
-	bigBeta0 := new(big.Float).Mul(newBeta0, new(big.Float).SetInt(common.Big2e64))
-	bigBeta0Int, _ := bigBeta0.Int(nil)
-
-	parent := hc.GetBlockByHash(block.ParentHash(common.PRIME_CTX))
-	if parent == nil {
-		return nil, errors.New("parent cannot be found")
-	}
-
-	minerDifficulty := hc.ComputeMinerDifficulty(parent)
 	// If parent is genesis, there is nothing to train
```

Comment:
This comment seems out of place because there is no genesis check in this function

Reply:
i agree, its an old commend will remove